server/controllers/recommendation/apRecordDecisionController.test.ts

```diff
@@ -9,19 +9,48 @@ jest.mock('../../monitoring/azureAppInsights')
 jest.mock('../../data/makeDecisionApiClient')
 
 describe('get', () => {
-  it('load record decision', async () => {
+  it('load record decision - RECALL', async () => {
     const res = mockRes({
       locals: {
         recommendation: {
           id: '1234',
+          spoRecallType: 'RECALL',
           spoRecallRationale: 'some reason',
         },
       },
     })
     const next = mockNext()
     await apRecordDecisionController.get(mockReq(), res, next)
 
-    expect(res.locals.page).toEqual({ id: 'apRecordDecision' })
+    expect(res.locals.page).toEqual({
+      id: 'apRecordDecision',
+      bodyText:
+        'This will be recorded as a contact in NDelius called ‘Management oversight- recall’. You cannot undo this.',
```

Comment:
Should be a space between oversight and hyphen.

Reply:
Just realised the `management insight` text is incorrect. This has been changed to `‘Management oversight - no recall`